content/handbook/security/corporate/end-user-services/supported-apps/reclaim-ai/_index.md

```
@@ -0,0 +1,121 @@
+stale: false
+---
+
+Reclaim.ai は、スケジューリングと時間管理の最適化を支援する AI 搭載のカレンダー管理および生産性ツールです。このプラットフォームはカレンダーシステムと統合し、フォーカスタイムの自動スケジューリング、タスクの管理、ミーティングスケジュールの最適化を行います。
+
+## セキュリティおよびコンプライアンス要件
+
+Reclaim.ai を使用する GitLab チームメンバーとして、あなたは以下のセキュリティ管理とコンプライアンス要件を理解し、遵守する責任があります:
+
+### 目次
+
+- [データ分類](#data-classification)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)